### ISSUE TITLE
Trees do not generate under stuff.

### ISSUE THREAD

**Issue by Azzyypaaras** [devs-immortal/The-Aether#424]
What it says on the can. Trees only generate on the topmost surface, this was not too major an issue before since it just meant areas covered by overhangs were clear of trees. However, aerclouds are stuff, and since they are generated via carver now, they are stuff that is placed down before the trees are, which causes large areas bare of trees, this is most egregious in thickets.
![image](https://user-images.githubusercontent.com/20349662/126850019-f5cf39fb-1275-4dbc-8fa2-f773bc23d3a3.png)
TLDR: we need to rework tree generation.

**Comment by Azzyypaaras** [comment]
For a matter of fact, this also applies to all groundcover, so uh, yeah this is bad.
Also, as an extra kick to the nuts, this makes it clear that boulders are badly distributed.

**Comment by Haven-King** [comment]
You need to use the `COUNT_MULTILAYER` decorator instead of the `COUNT_EXTRA`, where applicable. Features that don't have either of those decorators should probably also have the `COUNT_MULTILAYER` one.

**Comment by Azzyypaaras** [comment]
'ight who wants to change the one line in half the jsons

**Comment by Haven-King** [comment]
Trees aren't even in JSON 😛

**Comment by Azzyypaaras** [comment]
![hmmmm](https://user-images.githubusercontent.com/20349662/126852505-1fafee07-7f10-424f-a9af-0104a0fd6ce7.jpg)

**Comment by Azzyypaaras** [comment]
I'll fix trees up later then, not touching the json though :tater:

**Comment by SollyW** [comment]
Did someone say they want me to apply some regex to some json files?
